libraries/SITL/SIM_GPS.cpp

```diff
@@ -179,7 +185,7 @@ void GPS::send_ubx(uint8_t msgid, uint8_t *buf, uint16_t size)
 }
 
 /*
-  return GPS time of week in milliseconds
+  populate GPS time of week
```

Comment:
This function was not documented correctly. I fixed the docs, then added the functions I actually wanted to use below

Reply:
Comment is now correct, so this patch is wrong